lint-staged.config.js

```diff
@@ -23,10 +23,12 @@ const formatCss = [
 ];
 
 module.exports = {
-  '*.md': formatMarkdown,
   '{.github,.storybook,examples,materials,src}/**/*.md': formatMarkdown,
-  '*.js': formatJs,
+  './*.md': formatMarkdown,
+
   '{.storybook,examples,materials,scripts,src}/**/*.js': formatJs,
+  './*.js': formatJs,
```

Comment:
Can you explain the difference? We are applying `formatJs` to all top-level `js` files and to all `js` files in `.storybook,examples,materials,scripts,src`. Could we just apply it to all of them, as I'm not aware which ones  we need to actually exclude?

I have the same questions for the markdown-files.

Reply:
- `./*.js` matches only top level js files
- `{.storybook,examples,materials,scripts,src}/**/*.js` matches nested js files

I'm not sure what is confusing you...we're using the same setup in all repos

Reply:
I would like to find out which files (or if any) we need to exclude, because we could use `**/*.js` otherwise.